recipe/build.sh

```diff
@@ -4,4 +4,11 @@
 mv DESCRIPTION DESCRIPTION.old
 grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
 
+R_PACKAGE_NAME=${PKG_NAME:2}
+echo $R_PACKAGE_NAME
+
 $R CMD INSTALL --build .
+mkdir -p $PREFIX/R/library
+mv $PREFIX/lib/R/library/$R_PACKAGE_NAME $PREFIX/R/library/$R_PACKAGE_NAME
+
+echo "ln -s \${PREFIX}/R/library/$R_PACKAGE_NAME \${PREFIX}/lib/R/library/$R_PACKAGE_NAME" > $PREFIX/bin/.${PKG_NAME}-post-link.sh
```

Comment:
The above hack is because in windows the library is installed into `R/library` while in unix it's installed into `lib/R/library`